src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1285,8 +1285,7 @@ static bool addTriggeredSubscriptions_noCache
           }
           else
           {
-            trigs->stringFilterSet(stringFilterP);  // Clones the string filter
-            delete stringFilterP;
+            trigs->stringFilterSet(stringFilterP);
```

Comment:
Looking to the stringFilterSet() code it seems that now it is not cloning, so removing the comment is ok.

I have some doubt, but I'll use skype.

NTC